components/input/Radio.js

```diff
@@ -3,9 +3,9 @@ import PropTypes from 'prop-types';
 
 import { classnames } from '../../helpers/component';
```

Comment:
If you are not using `classnames` in this file, you need to remove this import.

Reply:
My bad, was because of my suggested change ^^

Reply:
@mmso 🔫 😄

Reply:
I'm fixing it no problem ;)

Reply:
Done https://github.com/ProtonMail/react-components/pull/231/commits/a1830410f71b97b164bd1d01a50c382eaae8b3b9